next/components/contact-list.tsx

```diff
@@ -0,0 +1,82 @@
+import NextImage from "next/image";
+import { useTranslation } from "next-i18next";
+
+interface Contact {
+  image: string;
+  name: string;
+  title: string;
+  phoneNumber: string;
+  email: string;
+  id: number;
+}
+
+const contacts: Contact[] = [
+  {
+    image: "/john_dean.jpg",
+    name: "John Dean",
+    title: "CFO",
+    phoneNumber: "+358445123456",
+    email: "john.dean@example.com",
+    id: 1,
+  },
+  {
+    image: "/charlie_dean.jpg",
+    name: "Charlie Dean",
+    title: "CTO",
+    phoneNumber: "+358445123458",
+    email: "charlie.dean@example.com",
+    id: 3,
+  },
+  {
+    image: "/jane_dean.jpg",
+    name: "Jane Dean",
+    title: "CEO",
+    phoneNumber: "+358445123457",
+    email: "jane.dean@example.com",
+    id: 2,
+  },
+];
+
+export function ContactList() {
+  const { t } = useTranslation();
+  return (
+    <section className="py-8">
+      <h2 className="text-heading-sm font-bold md:text-heading-md">
+        {t("contact")}
+      </h2>
+      <ul className="grid auto-rows-max grid-cols-1 justify-items-center gap-4 py-4 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4">
```

Comment:
`lg:grid-cols-4` with only 3 items means this is left-aligned on large screens, let's remove that class?

Reply:
On it.